src/Sensors.hpp

```diff
@@ -251,21 +259,22 @@ class Sensors {
 
     float getAltitude();
 
-    float getGas();
+    // float getGas();    // Posible colision con DFRobot Multigas
```

Comment:
I understand you. Maybe could be remove the gas method, or change it to real unit, a specific unit. Like, getCO2, getCO, getPM25.

Reply:
// float getGas();    // Posible colision con DFRobot Multigas
    
    float getNH3();
    
    float getCO();